cmd/podman/pod_ps.go

```diff
@@ -174,49 +170,20 @@ func podPsCmd(c *cliconfig.PodPsValues) error {
 
 	opts.Format = genPodPsFormat(c)
 
-	var filterFuncs []PodFilter
-	if c.Filter != "" {
-		filters := strings.Split(c.Filter, ",")
-		for _, f := range filters {
-			filterSplit := strings.Split(f, "=")
-			if len(filterSplit) < 2 {
-				return errors.Errorf("filter input must be in the form of filter=value: %s is invalid", f)
-			}
-			generatedFunc, err := generatePodFilterFuncs(filterSplit[0], filterSplit[1])
-			if err != nil {
-				return errors.Wrapf(err, "invalid filter")
-			}
-			filterFuncs = append(filterFuncs, generatedFunc)
-		}
-	}
-
 	var pods []*adapter.Pod
-	if c.Latest {
```

Comment:
I think we need to retain the `c.Latest` code - that's not a filter, but a separate flag

Reply:
Fixed. The flag is retained.